cmake_modules/commonSetup.cmake

```diff
@@ -243,6 +243,11 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   set(CMAKE_MODULE_PATH "${HPCC_SOURCE_DIR}/cmake_modules/")
 
   if(UNIX AND SIGN_MODULES)
+      execute_process(COMMAND bash "-c" "gpg --version | awk 'NR==1{print $3}'"
```

Comment:
Do any of the gpg changes affect the parameters that are used by eclcc / eclserver?